Sparkle/SUFileManager.m

```diff
@@ -178,9 +178,9 @@ - (BOOL)_removeXAttrWithAuthentication:(NSString *)name fromRootURL:(NSURL *)roo
         }
         return NO;
     }
-    
-    const char *xattrName = [name cStringUsingEncoding:NSASCIIStringEncoding];
-    if (xattrName == NULL) {
+
+    char xattrName[PATH_MAX] = {0};
+    if (![name getFileSystemRepresentation:xattrName maxLength:sizeof(xattrName)]) {
```

Comment:
Eh, [constness in C is broken](http://stackoverflow.com/questions/5055655/double-pointer-const-correctness-warnings-in-c) for array of strings :(

In that case maybe completely oldschool:

```c
char quarantine[] = "com.apple.quarantine";
```